### PR TITLE
Remove Needless Stuff

### PR DESCRIPTION
**Pull request by samusgray**
Removing needless things in an effort to make this a more focused resource for new team mates.

### CODE REVIEW

File: best-practices/README.md [hunk 1]
```diff
@@ -132,6 +132,7 @@ Testing
 * Use a [Fake] to stub requests to external services.
 * Use integration tests to execute the entire app.
 * Use non-[SUT] methods in expectations when possible.
+* Prefer data attribute selectors (ie `[data-test="foo"]`) over class selectors.
```
Comment (KrishnaKulkarni) [inline best-practices/README.md:135]: 👍

File: best-practices/README.md [hunk 2]
```diff
@@ -183,16 +184,11 @@ Background Jobs
 
 Email
 -----
+* Run copy changes by the Product team
+* Use a tool like [Mailcatcher] to look at each created or updated mailer view
```
Comment (KrishnaKulkarni) [inline best-practices/README.md:188]: 👍

File: best-practices/README.md [hunk 3]
```diff
@@ -206,16 +202,14 @@ JavaScript
 * Use the latest stable JavaScript syntax with a transpiler, such as [babel].
 * Include a `to_param` or `href` attribute when serializing ActiveRecord models,
   and use that when constructing URLs client side, rather than the ID.
+* Prefer data attribute selectors (ie `[data-behavior="foo"]`) over class selectors.
```
Comment (KrishnaKulkarni) [inline best-practices/README.md:205]: 👍

File: best-practices/README.md [hunk 4]
```diff
@@ -224,36 +218,29 @@ CSS
 
 Sass
 ----
-
+* Use SCSS syntax.
+* Use [BEM naming conventions].
```
Comment (KrishnaKulkarni) [inline best-practices/README.md:222]: Why is this in brackets?

File: best-practices/README.md [hunk 5]
```diff
@@ -224,36 +218,29 @@ CSS
...
 
 [autoprefixer]: https://github.com/postcss/autoprefixer
+[BEM naming conventions]: http://getbem.com/introduction/
 
 Browsers
 --------
 
-* Avoid supporting versions of Internet Explorer before IE10.
-
-Objective-C
```
Comment (KrishnaKulkarni) [inline best-practices/README.md:250]: Lol. Bye objective-c

File: how-to/README.md [hunk 6]
```diff
@@ -4,49 +4,7 @@
 
 See: [":rocket: __Domains, SSL, & YOU!__"](domains-ssl-you.md)
 
-## ... start a new Rails app?
+## ... start up the Healthify Rails app?
```
Comment (KrishnaKulkarni) [inline how-to/README.md:7]: 👍

File: protocol/communication/README.md [hunk 7]
```diff
@@ -3,7 +3,10 @@ Communication
 
 A guide for communicating within thoughtbot.
 
-Tickets
+Pivotal Tracker Stories
 -------
 
-* People assign themselves to tickets. When in doubt, do not assign a ticket to someone.
+* Software Engineers join the bi-weekly IPM to discuss the user stories (and chores, and bugs) for the week.
```
Comment (KrishnaKulkarni) [inline protocol/communication/README.md]: Bullet points 1 and 2 should be replaced with something like:
```
* Software Engineers are grouped into squads that work toward product themes over the course of a multi-week cycle. Squad leads organize ad hoc check-ins and planning meetings to scope out new user stories and set context for which people are most apt to work on them.
```

**Review by KrishnaKulkarni — APPROVED** [review]
Generally LGTM. I left like 2 actionable comments, but after fixing those merge at will!

File: best-practices/README.md (hunk 5 above)
Comment (samusgray) [reply best-practices/README.md:250]: Hahaha, for real.

File: protocol/communication/README.md (hunk 7 above)
Comment (samusgray) [reply protocol/communication/README.md]: Love it.

File: best-practices/README.md (hunk 4 above)
Comment (samusgray) [reply best-practices/README.md:222]: Oh, that should be a link to the BEM docs. Will add that.

Comment (samusgray) [reply best-practices/README.md:222]: Oh, it is linked! I used the advanced linking syntax, where it matches the string "[BEM...]" to the definition `[BEM naming conventions]: http://getbem.com/introduction/` below. Was just following the link pattern already established in this document.

**Pull request merged**